Library/Homebrew/formulary.rb

```diff
@@ -557,6 +655,12 @@ def self.loader_for(ref, from: nil)
     possible_alias = CoreTap.instance.alias_dir/ref
     return AliasLoader.new(possible_alias) if possible_alias.file?
 
+    if !CoreTap.instance.installed? &&
+       Homebrew::EnvConfig.install_from_api? &&
```

Comment:
I wonder whether we want a testing mode where you can always install from the API, even if the CoreTap is installed?

Reply:
This makes sense, though I'm not sure if it should be a part of this PR or not.

Reply:
Yeh, can punt on that being part of this PR if necessary. Just think it'd be a nice decoupling at some point.